pubspec.yaml

```diff
@@ -1,8 +1,8 @@
 name: pedantic
-version: 1.9.0
+version: 1.10.0-dev
 description: >-
   The Dart analyzer settings and best practices used internally at Google.
 homepage: https://github.com/dart-lang/pedantic
 
 environment:
-  sdk: '>=2.1.1-dev.0.0 <3.0.0'
+  sdk: '>=2.8.0-dev <3.0.0'
```

Comment:
We should change this to `<2.8.0` so that only 2.8.-dev sdks can select this release, per other comments.